parties/googleclient.go

```diff
@@ -112,7 +112,7 @@ func (g *GoogleOp) RequestTokens(cicHash string) ([]byte, error) {
 	}
 }
 
-func (g *GoogleOp) VerifyPKToken(pktJSON []byte, cosPk *ecdsa.PublicKey) (map[string]any, error) {
+func (g *GoogleOp) VerifyPKToken(pktJSON []byte, cosPk crypto.Signer) (map[string]any, error) {
```

Comment:
since cosPK is a public key, you should use `crypto.PublicKey` here rather than `crypto.Signer`

https://pkg.go.dev/crypto#PublicKey

Reply:
436ad42